src/backend/base/langflow/components/vectorsearch/CassandraSearch.py

```diff
@@ -26,20 +26,20 @@ def build_config(self):
             "embedding": {"display_name": "Embedding", "info": "Embedding to use"},
             "token": {
                 "display_name": "Token",
-                "info": "Authentication token for accessing Cassandra on Astra DB.",
+                "info": "Authentication token for Astra connections.",
                 "password": True,
             },
             "database_id": {
                 "display_name": "Database ID",
-                "info": "The Astra database ID.",
+                "info": "The Astra database ID. Used only for Astra connections.",
             },
             "table_name": {
                 "display_name": "Table Name",
                 "info": "The name of the table where vectors will be stored.",
```

Comment:
Doc says

A collection is a set of structured vector data. A table is the same but for non-vector data.

So I guess table is meant as non vector thing. 
From a Cassandra perspective there's only table so is fine but for Astra we talk about collection for thr vector store

Reply:
Hm, true. I was looking down at `Create a Table` section. In that case, I think you're right in that the `info` should use `collection` instead of `table`, though I will leave `table_name` as the parameter to match the cassio signatures. That sound good?